demos-europe/demosplan-core
# feature (refs T33554): fix procedure export

**Ticket:** https://yaits.demos-deutschland.de/T33554

Description: ZipExportService should return void, but return int.

### How to review/test
just check it in browser

Delete the checkbox if it doesn't apply/isn't necessary.
